src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/FancyZonesViewModel.cs

```diff
@@ -107,6 +109,8 @@ public FancyZonesViewModel(ISettingsRepository<GeneralSettings> settingsReposito
         private bool _openWindowOnActiveMonitor;
         private bool _spanZonesAcrossMonitors;
         private bool _restoreSize;
+        private bool _quickLayoutSwitch;
+        private bool _flashZones;
```

Comment:
Let's rename `_flashZones` to `_flashZonesOnQuickLayoutSwitch` for consistency with other places.